Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pulsar"
-version = "0.6.11"
+version = "0.6.12"
```

Comment:
This should also have updated in the `Cargo.lock` file but it doesn't appear to have done so. I think you can force it to update by running `cargo clippy`. This is what led to couple of release with `-2` at the end recently 🙂.